src/Admin/Translation/Controller/MachineTranslationAdminController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Admin\Translation\Controller;
+
+use App\Commands\Helpers\CommandHelper;
+use Sonata\AdminBundle\Controller\CRUDController;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+abstract class MachineTranslationAdminController extends CRUDController
```

Comment:
For better readability, I would recommend naming the class with an abstract prefix -> AbstractMach....Controller

Reply:
I could squeeze this into the next pr